code/bmpman/bmpman.cpp

```diff
@@ -1272,7 +1272,7 @@ bool bm_load_and_parse_eff(const char *filename, int dir_type, int *nframes, int
 	}
 	catch (const parse::ParseException& e)
 	{
-		mprintf(("BMPMAN: Unable to parse '%s'!  Error message = %s.\n", filename, e.what()));
+		nprintf(("BpMan","BMPMAN: Unable to parse '%s'!  Error message = %s.\n", filename, e.what()));
```

Comment:
This abbreviation should be "BmpMan", not "BpMan", surely.

Though backing up a bit, if these BmpMan messages are legit errors, why hide them?

Reply:
I would agree - if something critical happens, like an asset not being loaded, then that should be in the normal log.

Reply:
Latest pushes hopefully cover both these issues.